setup.cfg

```diff
@@ -35,6 +35,7 @@ universal = 1
 max-line-length = 99
 exclude = docs, .git, __pycache__, .ipynb_checkpoints
 ignore = W503
+extend-ignore = VNE001 # Single letter variable names are not allowed
```

Comment:
I see there is an `ignore` above this `extend`.
I think we should have only one of them: either we add `VN001` to the `ignore` list above, or we move the `V503` ignore here. This second option may be also adding other ignores (the defaults) behind the curtains, though, so I think the first one is preferable.

Reply:
Done. I planned to fix this in the base PR after everything was merged there, but I did it here instead.